Mathlib/MeasureTheory/Measure/OpenPos.lean

```diff
@@ -251,3 +253,35 @@ theorem measure_closedBall_pos (x : X) {r : ℝ≥0∞} (hr : r ≠ 0) : 0 < μ
 #align emetric.measure_closed_ball_pos EMetric.measure_closedBall_pos
 
 end EMetric
+
+section MeasureZero
+/-! ## Meagre sets and measure zero
+In general, neither of meagre and measure zero implies the other.
+- for all $α ∈ (0,1)$, there is a generalised Cantor set $C ⊆ [0,1]$ of measure `α`.
+Cantor sets are nowhere dense (hence meagre). Taking a countable union of fat Cantor sets whose
+measure approaches 1 even yields a meagre set of measure 1.
+- The set of Liouville numbers is a Lebesgue measure zero subset of ℝ, but is not meagre.
+
+However, with respect to a measure which is positive on non-empty open sets, *closed* measure
+zero sets are nowhere dense and σ-compact measure zero sets in a Hausdorff space are meagre.
+-/
+
+variable {X : Type*} [TopologicalSpace X] [MeasurableSpace X] [BorelSpace X] {s : Set X}
+  {μ : Measure X} [IsOpenPosMeasure μ]
+
+/-- A *closed* measure zero subset is nowhere dense. (Closedness is required: for instance, the
+rational numbers are countable (thus have measure zero), but are dense (hence not nowhere dense). -/
+lemma isNowhereDense_of_isClosed_null (h₁s : IsClosed s) (h₂s : μ s = 0) :
```

Comment:
If you rename your lemma like this:
```suggestion
lemma IsNowhereDense.of_isClosed_null (h₁s : IsClosed s) (h₂s : μ s = 0) :
```
then it'll be possible to write `.of_isClosed_null _ _` when Lean expects a proof of `IsNowhereDense _`. Same below.

Reply:
TIL, thanks! Changed.